superset/assets/src/dashboard/components/FilterIndicatorGroup.jsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { t } from '@superset-ui/translation';
+
+import FilterBadgeIcon from '../../components/FilterBadgeIcon';
+import FilterIndicatorTooltip from './FilterIndicatorTooltip';
+import FilterTooltipWrapper from './FilterTooltipWrapper';
+import { filterIndicatorPropShape } from '../util/propShapes';
+import { FILTER_LABELS } from '../../visualizations/FilterBox/FilterBox';
+
+const propTypes = {
+  indicators: PropTypes.arrayOf(filterIndicatorPropShape).isRequired,
+  setDirectPathToChild: PropTypes.func.isRequired,
+};
+
+class FilterIndicatorGroup extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    const { indicators, setDirectPathToChild } = this.props;
+    this.onClickIcons = indicators.map(indicator =>
+      setDirectPathToChild.bind(this, indicator.directPathToFilter),
+    );
+  }
+
+  render() {
+    const { indicators } = this.props;
+    const tooltipList = indicators.map((indicator, index) => (
+      <li key={index}>
```

Comment:
maybe make the key `indicator.name` instead?

Reply:
in the list there might have multiple filters have same name, which is a column name in different tables. now i used componentId, it should be unique all over dashboard.